cherryontech/jupiter-jumpers
# Update calendar and left bar styles

## Description

Updates the existing calendar to match the updated wireframe.

NOTE: Still waiting on the updated SVGs, so that will still be required in another PR.

## Related Issue

N/A


## Type of Changes

Style

## Acceptance Criteria


## Update Screenshots

### After
![Screenshot 2024-05-16 at 10 58 39 PM](https://github.com/cherryontech/jupiter-jumpers/assets/47403135/03430331-0734-4059-966e-06d30ff4236e)


## Testing Instructions

1. Check that adding a new log still works (I moved the button to the left sidebar)
2. Check that changing a day's mood works.
3. Check that you can switch months by clicking the two arrows on either side of the "May 2024"
4. Click "May 2024" and ensure you can change the month and year in that dropdown, and that it functions correctly.
5. Ensure the links in the menu work. 

## Learnings (Optional)

_Document any things you learned or 'gotchas' you experienced._
